resource-manager/src/test/java/org/snomed/module/storage/S3ClientIntegrationTest.java

```diff
@@ -1,89 +0,0 @@
-package org.snomed.module.storage;
-
-import org.junit.jupiter.api.Test;
-import software.amazon.awssdk.services.s3.model.Bucket;
-
-import java.util.List;
-
-import static org.junit.jupiter.api.Assertions.*;
-
-class S3ClientIntegrationTest extends IntegrationTest {
-    @Test
-    public void testCanReadBucketThatExists() {
```

Comment:
@pgwilliams this class was initially created to test integrating with localstack. It's no longer needed as it was essentially tests testing themselves.